react/FilterNavigator.js

```diff
@@ -263,6 +272,21 @@ const FilterNavigator = ({
               showClearByFilter={showClearByFilter}
               priceRangeLayout={priceRangeLayout}
             />
+            {showClearAllFilters && hasFiltersApplied && (
+              <div
+                className={`${applyModifiers(
+                  handles.filter__container,
+                  'clearFilters'
+                )} bb b--muted-4`}
+              >
+                <button
```

Comment:
Could you try to use the Button component from styleguide?
`import { Button } from 'vtex.styleguide'`